packages/manager/apps/dedicated/client/app/networkSecurity/network-security.module.js

```diff
@@ -0,0 +1,31 @@
+import angular from 'angular';
+import '@ovh-ux/ng-at-internet';
+import '@ovh-ux/ng-ui-router-breadcrumb';
+import 'ovh-api-services';
+import '@ovh-ux/ui-kit';
+
+import { ListLayoutHelper } from '@ovh-ux/manager-ng-layout-helpers';
+import { ApiV2ListHelper } from '../../../../../modules/ng-apiv2-helper';
+
+import component from './network-security.component';
+import routing from './network-security.routing';
+
+import scrubbing from './scrubbing-center';
+
+const moduleName = 'NetworkSecurity';
+
+angular
+  .module(moduleName, [
+    ListLayoutHelper.moduleName,
+    'ngAtInternet',
+    'ngUiRouterBreadcrumb',
+    'ovh-api-services',
```

Comment:
**nitpick:** `ovh-api-services` library is deprecated. Can you please remove this?
```suggestion
```